python/grass/gunittest/invoker.py

```diff
@@ -212,7 +213,8 @@ def _run_test_module(self, module, results_dir, gisdbase, location, timeout):
             if stdout is None:
                 stdout = ""
             if stderr is None:
-                stderr = f"Process has timed out in {timeout}s and produced no error output.\n"
+                stderr = f"Process has timed out in {timeout}s and produced no error \
+                    output.\n"
```

Comment:
This should be split using this syntax (the formatting of this example is off but that's not the point...):
```
s = ("this is a very "
      "long string"
     )